src/components/v2v3/V2V3Project/ManageNftsSection/RedeemNftsModal/RedeemNftCard.tsx

```diff
@@ -33,7 +33,7 @@ export function RedeemNftCard({
   onRemove,
   loading,
 }: {
-  nft: JB721DelegateToken
+  nft: Pick<JB721DelegateToken, 'address' | 'tokenId' | 'tokenUri'>
```

Comment:
good example of how we should strictly define types going forward. the compiler redlined this because we were only passing these props as `keys` in the subgraph query. previously we _could_ have tried using other JB721DelegateToken properties in the UI without any errors, but they would have been undefined